### PR TITLE
[xingyuyue/multi_ad_sample] add multivariate anomaly detetor sample f…

### PR DESCRIPTION
**Pull request by Xingyu-Yue**
…or v1.1

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/anomalydetector/azure-ai-anomalydetector/samples/sample_multivariate_detect.py [hunk 1]
```diff
@@ -19,200 +19,214 @@
...
-        data_feed = ModelInfo(start_time=start_time, end_time=end_time, source=self.data_source)
-        response_header = \
-            self.ad_client.train_multivariate_model(data_feed, cls=lambda *args: [args[i] for i in range(len(args))])[
-                -1]
-        trained_model_id = response_header['Location'].split("/")[-1]
+        while next_link != '':
+            r = self.ad_client.list_multivariate_model(skip=skip, top=10000)
+            next_link = r['nextLink'] 
+            model_list.extend(r['models'])
+            skip = skip + 10
```
Comment (rindesu) [inline sdk/anomalydetector/azure-ai-anomalydetector/samples/sample_multivariate_detect.py:55]: skip = skip + len(r['models'])

File: sdk/anomalydetector/azure-ai-anomalydetector/samples/sample_multivariate_detect.py [hunk 2]
```diff
@@ -19,200 +19,214 @@
...
 
-        # Use sample data to train the model
-        print("Training new model...(it may take a few minutes)")
-        data_feed = ModelInfo(start_time=start_time, end_time=end_time, source=self.data_source)
-        response_header = \
-            self.ad_client.train_multivariate_model(data_feed, cls=lambda *args: [args[i] for i in range(len(args))])[
-                -1]
-        trained_model_id = response_header['Location'].split("/")[-1]
+        while next_link != '':
+            r = self.ad_client.list_multivariate_model(skip=skip, top=10000)
```
Comment (rindesu) [inline sdk/anomalydetector/azure-ai-anomalydetector/samples/sample_multivariate_detect.py:52]: not recommended to set 'top' to such a large number. we can set it to 10 as an example

File: sdk/anomalydetector/azure-ai-anomalydetector/samples/sample_multivariate_detect.py [hunk 3]
```diff
@@ -19,200 +19,214 @@
...
+
+            ## Wait until the model is ready. It usually takes several minutes
+            model_status = None
+            model_info = None
+
+            while model_status != 'READY' and model_status != 'FAILED':
+                model_info = self.ad_client.get_multivariate_model(trained_model_id)
+                model_status = model_info['modelInfo']['status']
+                print("Model is", model_status)
+                time.sleep(1)
```
Comment (rindesu) [inline sdk/anomalydetector/azure-ai-anomalydetector/samples/sample_multivariate_detect.py:79]: time.sleep(30)? i think it is not necessary to check model status in such a high frequency

File: sdk/anomalydetector/azure-ai-anomalydetector/samples/sample_multivariate_detect.py [hunk 4]
```diff
@@ -19,200 +19,214 @@
...
-            r = self.ad_client.get_detection_result(result_id)
+            r = self.ad_client.get_batch_detection_result(result_id)
             print("Get detection result...(it may take a few seconds)")
 
-            while r.summary.status != DetectionStatus.READY and r.summary.status != DetectionStatus.FAILED:
-                r = self.ad_client.get_detection_result(result_id)
+            while r['summary']['status'] != 'READY' and r['summary']['status'] != 'FAILED':
+                r = self.ad_client.get_batch_detection_result(result_id)
+                print("Detection is", r['summary']['status'])
                 time.sleep(1)
```
Comment (rindesu) [inline sdk/anomalydetector/azure-ai-anomalydetector/samples/sample_multivariate_detect.py:122]: time.sleep(20)?

File: sdk/anomalydetector/azure-ai-anomalydetector/samples/sample_multivariate_detect.py [hunk 5]
```diff
@@ -19,200 +19,214 @@
...
+        "dataSchema": "MultiTable",
+        "startTime": "2021-01-02T00:00:00Z",
+        "endTime": "2021-01-02T05:00:00Z",
+        "displayName": "sample"
+    }
+    model_id = sample.train(train_body)
+
+    # Batch Inference
+    batch_inference_body = {
+        "dataSource": "{blobUrl}",
```
Comment (rindesu) [inline sdk/anomalydetector/azure-ai-anomalydetector/samples/sample_multivariate_detect.py:182]: the params 'startTime' and 'endTime' is related to sample data here. Will we provide a fixed sample data to user for them to try e2e successfully?

File: sdk/anomalydetector/azure-ai-anomalydetector/CHANGELOG.md [hunk 6]
```diff
@@ -12,6 +12,24 @@
 
 - Python 2.7 is no longer supported. Please use Python version 3.6 or later.
 
+## 3.0.0b6 (2022-11-01)
+  **Features**
+  - Added operation AnomalyDetectorClientOperationsMixin.xxx
```
[the remark this answers is not in the document]
Comment (minkefusiji) [reply sdk/anomalydetector/azure-ai-anomalydetector/CHANGELOG.md]: Resolved.

File: shared_requirements.txt [hunk 7]
```diff
@@ -542,3 +542,7 @@ opentelemetry-sdk<2.0.0,>=1.5.0,!=1.10a0
 #override azure-core-experimental azure-core<2.0.0,>=1.25.0
 #override azure-developer-devcenter isodate<1.0.0,>=0.6.1
 #override azure-developer-devcenter azure-core<2.0.0,>=1.24.0
+#override azure-ai-anomalydetector isodate<1.0.0,>=0.6.1
+#override azure-ai-anomalydetector azure-core<2.0.0,>=1.24.0
+#override azure-ai-anomalydetector typing_extensions>=4.3.0
```
Comment (msyyc) [inline shared_requirements.txt:547]:
```suggestion
#override azure-ai-anomalydetector typing-extensions>=4.3.0; python_version<'3.8.0'
```
It shall keep consistency with `setup.py`:

File: shared_requirements.txt [hunk 8]
```diff
@@ -542,3 +542,7 @@ opentelemetry-sdk<2.0.0,>=1.5.0,!=1.10a0
...
+#override azure-ai-anomalydetector typing-extensions>=4.3.0
```
Comment (msyyc) [inline shared_requirements.txt:548]:
```suggestion
```
duplicated

File: sdk/anomalydetector/azure-ai-anomalydetector/README.md [hunk 9]
```diff
@@ -1,100 +1,43 @@
-# Microsoft Azure SDK for Python
 
-This is the Microsoft Azure Cognitive Services Anomaly Detector Client Library.
-This package has been tested with Python 3.6+.
+# Cognitive Services Anomaly Detector client library for Python
+<!-- write necessary description of service -->
```
Comment (msyyc) [inline sdk/anomalydetector/azure-ai-anomalydetector/README.md:3]: Please update `readme.md` for better usage experience of customer.

File: sdk/anomalydetector/azure-ai-anomalydetector/swagger/README.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
+### Settings
+
+```yaml
+input-file: 
+ - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/UnivariateAnomalyDetector.json
+ - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/MultivariateAnomalyDetector.json
+output-folder: ../azure-ai-anomalydetector-new
```
Comment (msyyc) [inline sdk/anomalydetector/azure-ai-anomalydetector/swagger/README.md:7]:
```suggestion
output-folder: ../azure-ai-anomalydetector
```

File: sdk/anomalydetector/ci.yml [hunk 11]
```diff
@@ -20,6 +20,7 @@ pr:
     - hotfix/*
     - release/*
     - restapi*
+    - xingyuyue/multi_ad_sample
```
Comment (msyyc) [inline sdk/anomalydetector/ci.yml:23]: Please remember to delete it before merge into main

**Review by annatisch — APPROVED** [review]

File: sdk/anomalydetector/azure-ai-anomalydetector/swagger/README.md [hunk 12]
```diff
@@ -0,0 +1,17 @@
+### Settings
+
+```yaml
+input-file: 
+ - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/UnivariateAnomalyDetector.json
+ - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/MultivariateAnomalyDetector.json
+output-folder: ../azure-ai-anomalydetector
```
Comment (msyyc) [inline sdk/anomalydetector/azure-ai-anomalydetector/swagger/README.md:7]:
```suggestion
output-folder: ../
```
@Xingyu-Yue output-folder is not right

File: sdk/anomalydetector/azure-ai-anomalydetector/CHANGELOG.md [hunk 13]
```diff
@@ -1,16 +1,22 @@
...
-## 3.0.0b6 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+## 3.0.0b6 (2022-11-01)
```
Comment (msyyc) [inline sdk/anomalydetector/azure-ai-anomalydetector/CHANGELOG.md:3]: Before merge it, please update date to your target release date.

**Review by lmazuel — APPROVED** [review]

**Pull request merged**